modules/websocket/wsl_client.cpp

```diff
@@ -327,10 +327,10 @@ uint16_t WSLClient::get_connected_port() const {
 Error WSLClient::set_buffers(int p_in_buffer, int p_in_packets, int p_out_buffer, int p_out_packets) {
 	ERR_FAIL_COND_V_MSG(_connection.is_valid(), FAILED, "Buffers sizes can only be set before listening or connecting.");
 
-	_in_buf_size = nearest_shift(p_in_buffer - 1) + 10;
-	_in_pkt_size = nearest_shift(p_in_packets - 1);
-	_out_buf_size = nearest_shift(p_out_buffer - 1) + 10;
-	_out_pkt_size = nearest_shift(p_out_packets - 1);
+	_in_buf_size = nearest_shift(MAX(0, p_in_buffer - 1)) + 10;
+	_in_pkt_size = nearest_shift(MAX(0, p_in_packets - 1));
+	_out_buf_size = nearest_shift(MAX(0, p_out_buffer - 1)) + 10;
+	_out_pkt_size = nearest_shift(MAX(0, p_out_packets - 1));
```

Comment:
@Faless Should we error out instead of parameters are `< 1`?

Reply:
Yes, we should. Actually, we should error out if parameters are `< 2` (for `p_*_buffer` it might make sense something like `< 512` or `< 1024`)